bot.py

```
@@ -913,6 +914,69 @@ async def sum_hr_slash(interaction: discord.Interaction, hours: int):
+@discord.app_commands.describe(
+    role="The role you want to join (voice-gang or live-gang)"
+)
+@discord.app_commands.choices(role=[
+    discord.app_commands.Choice(name="Voice Gang - Get pinged for voice chats", value="voice-gang"),
+    discord.app_commands.Choice(name="Live Gang - Get notified when streams start", value="live-gang")
+])
+async def join_role(interaction: discord.Interaction, role: discord.app_commands.Choice[str]):
+    """Allow users to self-assign to approved roles"""
+    await interaction.response.defer(ephemeral=True)
+
+    success, message = await RoleManager.add_role_to_member(
```

Ruff (0.14.3)

Unpacked variable `success` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)